specification.markdown

```diff
@@ -235,7 +235,7 @@ Within an EBML Schema the `<element>` uses the following attributes to define an
 | minver         | No       | The `minver` (minimum version) attribute stores a non-negative integer that represents the first version of the docType to support the element. If the `minver` attribute is not used it is assumed that the element has a minimum version of "1". |
 | maxver         | No       | The `maxver` (maximum version) attribute stores a non-negative integer that represents the last or most recent version of the docType to support the element. If the `maxver` attribute is not used it is assumed that the element has a maximum version equal to the value stored in the `version` attribute of <EBMLSchema>. |
 
-The `<element>` nodes shall contain a description of the meaning and use of the EBML Element stored within one or many `<documentation>` sub-elements. The `<documentation>` sub-element may use a `lang` attribute which may be set to the RFC 5646 value of the language of the element's documentation.
+The `<element>` nodes shall contain a description of the meaning and use of the EBML Element stored within one or many `<documentation>` sub-elements. The `<documentation>` sub-element may use a `lang` attribute which may be set to the RFC 5646 value of the language of the element's documentation. The `<documentation>` sub-element may use a `type` attribute to distinguish the meaning of the documentation. Recommended values for the `<documentation>` sub-element's `type` attribute include: definition, rationale, usage notes, and references.
```

Comment:
I'd prefer to have the four possible values for the `type` attribute to be placed in `code tags`, too, as that's supposed to be an enumeration of fixed strings.